File: viviane013/programacao_basica/Questao14.java
import java.util.Scanner;
public class Questao14 {
        public static void main(String[] args) {
            Scanner s;
            int dias;
            int anos;
            int dia;
            int meses;
            int d;

            s = new Scanner(System.in);

            System.out.println("Calculador de anos,meses e anos");
            
            System.out.print("Diga-me o numero: ");
            dias = s.nextInt();


            anos = dias / 360;
            dia = dias - (anos * 360);
            meses = dia / 30;
            d = dia - (meses * 30);

            System.out.println("Numero informado em dias: " + d);
            System.out.println("Numero informado em meses: " + meses);
            System.out.println("Numero informado em anos: " + anos);
    }
}

File: viviane013/programacao_basica/questao14.py
print("Calculador de anos,meses e anos ")
dias = int(input("Diga-me o numero: "))
anos = dias // 360
dia = dias - (anos * 360)
meses = dia // 30
d = dia - (meses * 30)

print("numero informado em dias: " + str(d))
print("numero informado em meses: " + str(meses))
print("numero informado em anos: " + str(anos))



